reqhandler/fields.go

```diff
@@ -0,0 +1,33 @@
+package reqhandler
+
+type FieldName string
```

Comment:
We have a Header struct, which groups these names. Because we have other field names (Question, etc) we can have constant Field Names to represent this (it isn't good style in Golang to pass around strings).